app/components/concept-page/content.hbs

```diff
@@ -60,6 +60,17 @@
               </div>
             {{/animated-if}}
           </div>
+
+          <div class="py-12 border-t" {{did-insert this.handleShareConceptContainerInserted}} data-test-share-concept-container>
```

Comment:
Let's place this above `nextConcept` please, feels more structured/coherent to have the order be "share feedback on current concept", "share current concept", "move on to next concept". 

And we don't need `animated-if` for `nextConcept` above - that value isn't going to change while the user is viewing a page (it's loaded when the concept page is loaded). A plain `#if` should do I think